app/models/ems_refresh/refreshers/container_refresher_mixin.rb

```diff
@@ -0,0 +1,38 @@
+module EmsRefresh
+  module Refreshers
+    module ContainerRefresherMixin
+      def kubernetes_fetch_inventory(client)
+        fetch_entities(client, KUBERNETES_ENTITIES)
+      end
+
+      def openshift_fetch_inventory(client)
+        fetch_entities(client, OPENSHIFT_ENTITIES)
+      end
```

Comment:
> But, that goes back to my original point that I think this could probably be handled through inheritance or through some builder pattern, since Openshift == Kubernetes + Extra Stuff. But, @simon3z and I have a difference of opinion there.

@blomquisg if `container_refresher_mixin.rb` was called `kubernetes_refresher_mixin.rb` (containing only kubernetes specific code for the refresh) and it was included by both the Kubernetes and OpenShift refresher, wouldn't it be as OpenShift (refresher) == Kubernetes (refresher) + Extra Stuff?
...and even better because composition is preferable vs inheritance as it scopes better what it shared and what's not.